chall02/szavarce.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+import sys
+codigo_morse = {
+  "A": ".-", "B": "-...", "C": "-.-.", "D": "-..", "E": ".", "F": "..-.", "G": "--.", "H": "....", "I": "..", "J": ".---", "K": "-.-", "L": ".-..", "M": "--", "N": "-.", "O": "---", "P": ".--.", "Q": "--.-", "R": ".-.", "S": "...", "T": "-", "U": "..-", "V": "...-", "W": ".--", "X": "-..-", "Y": "-.--", "Z": "--..", " ": " "
+}
+resultado = ""
+if len(sys.argv) != 2:
+  print("usage: ./szavarce.py <a-zA-Z string>")
+  exit()
+
+for c in sys.argv[1]:
+  if c.upper() in codigo_morse:
+    resultado += codigo_morse[c.upper()]
+  else:
+    print("usage: ./szavarce.py <a-zA-Z string>")
+    exit()
```

Comment:
You could have made this more pythonic with those oneliners:
```suggestion
if not sys.argv[1].replace(" ", "").isalpha():
    print("usage: ./szavarce.py <a-zA-Z string>")
    exit()
resultado = "".jon(codigo_morse[c] for c in sys.argv[1])
```